openedx/core/djangoapps/content_staging/views.py

```diff
@@ -110,6 +109,8 @@ def post(self, request):
                 version_num = None
 
             elif isinstance(course_key, LibraryLocatorV2):
+                from openedx.core.djangoapps.content_libraries import api as lib_api
+
```

Comment:
Any reason for moving this here.

Reply:
https://github.com/openedx/edx-platform/blob/47d253a1fee52b76b7dd583e63c0bc8074fb8a7e/openedx/core/djangoapps/content_staging/views.py#L85-L90

We should stop importing the `content_libraries` here in the future, but we probably need to import the other way around. I moved it here to avoid circular imports.